core/src/main/java/org/infinispan/xsite/BaseBackupReceiver.java

```diff
@@ -123,6 +125,20 @@ public Object visitReplaceCommand(InvocationContext ctx, ReplaceCommand command)
       }
 
       @Override
+      public Object visitComputeCommand(InvocationContext ctx, ComputeCommand command) throws Throwable {
+         log.tracef("Processing a remote compute %s", command);
```

Comment:
I was meaning maybe we should have a log.tracef for all of the visitXCommand methods. Maybe @pruivo thinks this is helpful?